guides/v2.1/howdoi/checkout/checkout_new_field.md

```diff
@@ -52,6 +52,8 @@ $customField = [
     'filterBy' => null,
     'customEntry' => null,
     'visible' => true,
+    // value is used to set a default value of the attribute
+    'value' => ''
```

Comment:
Hi @VitaliyBoyko. Could you set the inline commend on the same line as the value, please? Also, adjust the comment slightly by adding the "value field" clarification
```suggestion
    'value' => '' // value field is used to set a default value of the attribute
```

Reply:
@rogyar
Done.
Thanks!